core/src/integration-test/java/com/scalar/db/storage/cosmos/CosmosConditionalMutationIntegrationTest.java

```diff
@@ -37,6 +37,9 @@ protected Map<String, String> getCreationOptions() {
   protected List<OperatorAndDataType> getOperatorAndDataTypeListForTest() {
```

Comment:
This is a minor suggestion. I think we can modify this in the following way, ensuring we won't need any adjustments for similar changes in the future:

```Java
  @Override
  protected List<OperatorAndDataType> getOperatorAndDataTypeListForTest() {
    return super.getOperatorAndDataTypeListForTest().stream()
        .filter(
            operatorAndDataType -> {
              // Cosmos DB only supports the 'equal' and 'not equal' and 'is null' and 'is not null'
              // conditions for BLOB type
              if (operatorAndDataType.getDataType() == DataType.BLOB) {
                return operatorAndDataType.getOperator() == Operator.EQ
                    || operatorAndDataType.getOperator() == Operator.NE
                    || operatorAndDataType.getOperator() == Operator.IS_NULL
                    || operatorAndDataType.getOperator() == Operator.IS_NOT_NULL;
              }

              return true;
            })
        .collect(Collectors.toList());
  }
```